python/tank/bootstrap/manager.py

```diff
@@ -59,6 +59,8 @@ def __init__(self, sg_user=None):
                         authentication, simply construct an explicit user object
                         and pass it in.
         :type sg_user: :class:`~sgtk.authentication.ShotgunUser`
+        :param bool allow_config_overrides: Whether to allow the config override
+            environment variable to affect the results of configuration resolutions.
```

Comment:
minor: it would be great to add an `..info:` box or similar to briefly explain the use case when this can be helpful.